unit_testing/test_sct_compute_mtsat.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+from __future__ import print_function, absolute_import
```

Comment:
Needed?

Reply:
no, fixed in e496a39d09d17fd9dd8816fd62e5cd85d4199fa6